### PR TITLE
Feature/iteration2loginpage

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

- Log in button not bringing up a form for a user to log in

## What is the new behavior?

- A form is brought up and a user can log in and and the homepage will rerender

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

Next is getting the users ratings to display!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,21 +2,38 @@ import React from 'react';
...
   constructor() {
     super();
     this.state = {
       movies: null,
       isLoading: false,
-      error: null
+      error: null,
+      // isLoggedIn: false,
+      // form: false,
+      logInMethod: this.logIn,
```
Comment (carflor) [inline src/App.js:16]: is this form:false supposed to be commented out? are we able to render the login without it?

File: src/Components/LogInForm.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.main-page {
```
Comment (carflor) [inline src/Components/LogInForm.css:1]: Looking good!

File: src/Components/LogInForm.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
```
Comment (carflor) [inline src/Components/LogInForm.js:1]: awesome job with the submit method.

File: src/App.js (hunk 1 above)
Comment (Billwilke42) [reply src/App.js:16]: Yes we are, was playing around and commenting stuff out to get rid of extraneous code

**Pull request merged**